packages/verification-vue-component/src/components/Verification/items/cards/SigningCard.vue

```diff
@@ -0,0 +1,237 @@
+<template>
```

Comment:
The SigningCard shares quite a lot of code with the VerificationCard. I think these should be combined, at least in the future. If it's fully event based, then there are only some minor differences how it looks between the different events.